study-builder/tenants/circadia/tenant.yaml

```diff
@@ -0,0 +1,50 @@
+tenant:
+  default_directory: Username-Password-Authentication
+
+rules:
+  - name: Register User in Pepper
+    script: ../register-user-in-pepper.js
+    stage: login_success
+    enabled: true
+    order: 1
+
+pages:
+  - name: login
+    enabled: true
+    html: ./pages/login.html
+  - name: password_reset
+    enabled: true
+    html: ./pages/password_reset.html
+
+emailTemplates:
+  - template: welcome_email
+    enabled: true
+    syntax: liquid
+    from: '##FROM_EMAIL##'
+    subject: Account Created
+    body: emails/welcome_email.html
+
+clients:
+  - name: Circadia Dev
```

Comment:
We're going to run this `tenant.yaml` for all the different environments (including production!) to setup Auth0. I recommend naming this client `circadia-angular-client` or similar.